Libplanet/Blockchain/Renderers/LoggedActionRenderer.cs

```diff
@@ -44,7 +44,7 @@ public class LoggedActionRenderer<T> : LoggedRenderer<T>, IActionRenderer<T>
         public LoggedActionRenderer(
             IActionRenderer<T> renderer,
             ILogger logger,
-            LogEventLevel level = LogEventLevel.Debug
+            LogEventLevel level = LogEventLevel.Information
```

Comment:
Wouldn't this prevent debug messages from being logged as stdout? I might have misunderstood but I thought we were logging all the debug messages on the stdout but filter info logs to CW.

Reply:
I suspect this breaks tests. 🤔